examples/raysgd_nyctaxi.py

```diff
@@ -38,12 +43,16 @@
 # Split data into train_dataset and test_dataset
 train_df, test_df = random_split(data, [0.9, 0.1], 0)
 features = [field.name for field in list(train_df.schema) if field.name != "fare_amount"]
-# Convert spark dataframe into ML Dataset
-train_dataset = RayMLDataset.from_spark(train_df, num_executors, 32)
-test_dataset = RayMLDataset.from_spark(test_df, num_executors, 32)
+
+# Convert spark dataframe into ray Dataset
+train_dataset = ray.data.from_spark(train_df, parallelism = num_executors)
+test_dataset = ray.data.from_spark(test_df, parallelism = num_executors)
 # Then convert to torch datasets
-train_dataset = train_dataset.to_torch(feature_columns=features, label_column="fare_amount")
-test_dataset = test_dataset.to_torch(feature_columns=features, label_column="fare_amount")
+feature_dtype = [torch.float] * len(features)
+train_dataset = train_dataset.to_torch(feature_columns=features, label_column="fare_amount",
+                                               label_column_dtype=torch.float, feature_column_dtypes=feature_dtype, batch_size=64)
+test_dataset = test_dataset.to_torch(feature_columns=features, label_column="fare_amount",
+                                             label_column_dtype=torch.float, feature_column_dtypes=feature_dtype, batch_size=64)
```

Comment:
We should get shards of the dataset inside the train_func, check [here](https://docs.ray.io/en/latest/train/user_guide.html#train-datasets).